30번까지 한글화하였습니다.

jihoonl opened the pull request:
영문 판은 재 수정후에 다시 올리도록 하겠습니다. 한글화를 30번까지 완료하였습니다. 29번이 어색한데 확인 부탁드립니다.

rebase하는 과정에서 왠지 모르지만 1번이 추가되었네요.

---

```
캐리지 리턴의 변화로 파일 전체가 변환되었습니다. 차이를 보고 싶은데 어렵네요.

*잘 안들렸던 부분 목록*

27
00:01:44,810 --> 00:01:48,618
and the K neighbor classifiers  and I talked about the secret India said

29
00:01:52,938 --> 00:01:58,438
the idea of this approach that I termed paratroop approach which is really that

39
00:02:32,240 --> 00:02:36,530
along the columns and we have some classes say 10 classes and score 10

74
00:04:50,949 --> 00:04:55,348
loss equal two stuff and I'm going to interpret this loss now for Easter that

147
00:10:13,389 --> 00:10:28,000
the score constraints. (Student questions) So you are saying if there\'92re too many classes dilute the loss. (Student mumbled) So you're right in that the

148
00:10:28,000 --> 00:10:33,870
absolute value of the loss will be lower. (Student mumbled)

149
00:10:33,870 --> 00:10:37,879
So a constant factor why
```

jazzsaxmafia committed:
Update convolutional-networks-korean.md

rollis committed:
Lecture1 - part 61~100 (out of 715) en / ko

rollis committed:
Lecture1 - part 61~100 (out of 715) en / ko

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

joonkuim committed:
Update optimization-1.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

Commit:
160번까지 듣기평가 완료. 부족한 부분이 좀 있습니다.

Commit:
160번까지 듣기평가 완료. 부족한 부분이 좀 있습니다.

jazzsaxmafia committed:
Update convolutional-networks-korean.md

Commit:
Normalization Draft

OkminLee committed:
Update classification.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

younggeunchoi committed:
Update neural-networks-3.md

younggeunchoi committed:
Update neural-networks-3.md

PR 연습 겸 업데이트 해봅니다.
총 400라인 정도의 문서인데 오늘 50라인 정도 마쳤습니다. 틈틈이 또 업데이트 하겠습니다.

jiunbae committed:
assignment#1/features.ipynb

jiunbae committed:
remove etc

myungsub committed:
Merge pull request #22 from rollis/master

Lecture1 - part 61~100 (out of 715) en / ko

myungsub commented:
수고하셨습니다! 좀이따 적어주신 부분 확인 후 합칠게요!1

jiunbae committed:
Update assignment1/features.ipynb

Commit:
Dropout까지 완료

rollis committed:
Lecture1 - part 101~120 (out of 715) en / ko

Commit:
30번까지 완료

Commit:
30번까지 완료

jihoonl commented:
30번까지 한글화 완료했습니다. 한글화중 29번이 미흡합니다. 확인 부탁드려요.

Commit:
reverting Lect3_en.srt.

Commit:
reverting Lect3_en.srt.

younggeunchoi committed:
github desktop 연습

myungsub commented:
잘 안들린다고 하신 부분 확인해 보았습니다.
뉴럴네트워크를 테스트 해볼때 자주 사용하는 CIFAR-10 dataset의 발음이 "see-far ten" 이렇게 되어서 헷갈리신 부분이 많이 있는 것 같아요!

27
00:01:44,810 --> 00:01:48,618
and the K neighbor classifier.  And I talked about the CIFAR-10 dataset

29
00:01:52,938 --> 00:01:58,438
the idea of this approach that I termed parametric approach which is really that

39
00:02:32,240 --> 00:02:36,530
along the columns and we have some classes say 10 classes in CIFAR-10

74 (100% 확실치는 않네요..)
00:04:50,949 --> 00:04:55,348
loss equal to.. stuff, and I'm going to interpret this loss now for ease to that

147
00:10:13,389 --> 00:10:28,000
the score constraints. (Student questions) So you are saying if there were too many classes that would guide the loss. (Student mumbled) So you're right in that the

148
00:10:28,000 --> 00:10:33,870
absolute value of the loss will be get lowered. (Student mumbled)

149 (여기도 100% 확실치는..ㅠ)
00:10:33,870 --> 00:10:37,879
So it would be a constant factor of y?

jiunbae committed:
Update .gitignore (.ipynb_checkpoints)

Update Translate

myungsub committed:
Merge pull request #28 from MaybeS/master

assignment#1/features.ipynb

myungsub committed:
Merge pull request #29 from rollis/master

Lecture1 - part 101~120 (out of 715) en / ko

joonkuim committed:
Update optimization-1.md

rollis committed:
Lecture1 - part 121~140 (out of 715) en / ko

gnujoow committed:
초안작성 완료

dhl8282 committed:
Translate understanding-cnn.md file test (#33)

Test commit for translating understanding-cnn.md file

gnujoow committed:
Terminal.md :: Importance Note

gnujoow committed:
no message

gnujoow committed:
python-tutorial.md : draft

gnujoow committed:
no message

gnujoow committed:
Update ipython-tutorial.md

joonkuim committed:
Update optimization-1.md

joonkuim committed:
Update optimization-1.md

optimization-1.md 끝까지 번역해봤습니다. 지난 번 PR한거(아직 승인 안 하신 듯)에서 큰 오역(졸았나...)을 발견해서 그것도 수정했습니다.

joonkuim committed:
Update optimization-1.md

Glossary에서 Batch가 배치로 되어 있는 것을 뒤늦게 발견해서 업데이트했습니다.

jiunbae committed:
Update assignment1/knn.ipynb

rollis committed:
Lecture1 - part 141~160 (out of 715) en / ko

myungsub commented:
파일이 변환되면서 merge conflict가 나네요ㅠㅠ
마스터 브랜치에서 다시 pull한 뒤에 PR에 새로 커밋해주시면 감사하겠습니다!

myungsub committed:
Merge pull request #31 from rollis/master

Lecture1 - part 121~140 (out of 715) en / ko

myungsub committed:
Merge pull request #21 from jazzsaxmafia/master

Update convolutional-networks-korean.md

myungsub committed:
Merge pull request #25 from salopge/neural-network-2

neural-networks-2.md

OkminLee committed:
Update classification.md

OkminLee committed:
Update classification.md

OkminLee committed:
Update classification.md

jazzsaxmafia committed:
Update convolutional-networks-korean.md

j-min committed:
Lecture 10 번역 중간 업데이트

~33:33 까지 번역했습니다. 한동안 집중을 못했었는데 최대한 빨리 끝내도록 할게요!

Commit:
Update optimization-1.md

모수=>파라미터
점수함수=>스코어함수 (오류 하나)
파이썬 코드 코멘트 번역 안 했던거 추가 번역.

Commit:
Update optimization-1.md

Commit:
Update optimization-1.md

OkminLee committed:
Update classification.md

myungsub committed:
Merge pull request #37 from j-min/patch-2

Lecture 10 번역 중간 업데이트

myungsub committed:
Merge pull request #34 from gnujoow/master

Terminal.md & IPython-turoial.md 초안 작성완료

myungsub committed:
Merge pull request #30 from stats2ml/patch-4

Update optimization-1.md

jiunbae committed:
Update cross-validate

jiunbae committed:
Remove Original lines

myungsub committed:
Merge pull request #36 from MaybeS/master

Update assignment1/knn.ipynb

myungsub committed:
Merge pull request #26 from OkminLee/patch-2

Update classification.md

myungsub committed:
translate main page

rollis committed:
Lecture1 - part 161~170 (out of 715) en / ko

myungsub committed:
Add Acknowledgement

myungsub committed:
Add video info & acknowledgement

jazzsaxmafia committed:
Update convolutional-networks.md

gnujoow committed:
ipython-tutorial의 띄어쓰기 오류 수정

gnujoow committed:
terminal-tutorial.md 띄어쓰기 오류 정정

gnujoow committed:
Update terminal-tutorial.md

myungsub committed:
Merge pull request #40 from gnujoow/patch-2

ipython-tutorial의 띄어쓰기 오류 수정

myungsub committed:
Merge pull request #41 from gnujoow/patch-3

terminal-tutorial.md 띄어쓰기 오류 수정

myungsub committed:
Merge pull request #38 from rollis/master

Lecture1 - part 161~170 (out of 715) en / ko

younggeunchoi committed:
Add files via upload

생각보다 작업이 더뎌서 죄송합니다. 전체 7000단어 중 1200단어 가량 (1/6) 마쳤습니다.

younggeunchoi committed:
Delete neural-networks-3-kr.md

myungsub committed:
Update optimization-1.md

stats2ml committed:
Update index.html

optimization-1 부분 소제목/키워드 업데이트.

myungsub committed:
Merge pull request #43 from stats2ml/patch-6

Update index.html

myungsub committed:
Merge pull request #27 from ygchoistat/master

update neural-networks-3.md

myungsub committed:
Add acknowledgement

myungsub committed:
fix typo

Commit:
Vanila Dropout

kirumang committed:
first_commit_during_translate

Commit:
fix typo

myungsub committed:
Merge pull request #45 from kirumang/master

강의 노트 번역 중간 공유

Commit:
정규화까지 완료

stats2ml committed:
Update neural-networks-1.md

myungsub committed:
Merge pull request #46 from salopge/neural-network-2

Neural network 2

myungsub committed:
Merge pull request #47 from stats2ml/patch-7

Update neural-networks-1.md

myungsub committed:
Update glossary.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

jazzsaxmafia committed:
Update convolutional-networks.md

myungsub committed:
initial progress bar

myungsub committed:
add progress bars

jazzsaxmafia committed:
영어 원문을 안 지운 문단이 있어서 지움

jihoonl committed:
resolve conflicts

jihoonl commented:
다시 PR 하겠습니다.
